Formula/siege.rb

```diff
@@ -1,8 +1,8 @@
 class Siege < Formula
   desc "HTTP regression testing and benchmarking utility"
   homepage "https://www.joedog.org/siege-home/"
-  url "http://download.joedog.org/siege/siege-4.0.9.tar.gz"
-  sha256 "1dbe15860569e7becedfefca6cd60d1fdba2eed281098e51718ca70d2575f277"
+  url "http://download.joedog.org/siege/siege-4.1.0.tar.gz"
+  sha256 "367927503856620e21328e70d091c47f869c29fe1fdb724cb7291cd48190d111"
   license "GPL-3.0"
```

Comment:
Bit confusing, the `COPYING` file specifies the GPLv3, but the file headers are 
```c
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
```
So I think that means it's technically 3 or later?
```suggestion
  license "GPL-3.0-or-later"
```